src/couch-config.js

```diff
@@ -0,0 +1,151 @@
+import * as utils from './utils.js';
+import log from 'npmlog';
+import Wreck from 'wreck';
+import Promise from 'bluebird';
+import prettyjson from 'prettyjson';
+import nmo from './nmo.js';
+import isonline, { getClusterUrls } from './isonline.js';
+
+export function cli (cmd, cluster, section, key, value) {
+  return new Promise((resolve, reject) => {
+
+    if (!cmd || !cluster || !exports[cmd]) {
+      const err = new Error('Usage: nmo couch-config add/set <cluster> ...');
```

Comment:
one small thing, can you change the usage message that it explains both commands, like in https://github.com/apache/couchdb-nmo/blob/35982a557a61ef04bb2ff66302dfe29be2e78fea/src/cluster.js#L24